### PR TITLE
Camel Case the world

### PR DESCRIPTION
**Pull request by kblok**

### CODE REVIEW

File: lib/PuppeteerSharp.Tests/CSSCoverageTests/CSSCoverageTests.cs [hunk 1]
```diff
@@ -3,6 +3,7 @@
 using System.Text.RegularExpressions;
 using System.Threading.Tasks;
 using Newtonsoft.Json;
+using Newtonsoft.Json.Serialization;
```
Comment (Meir017) [inline lib/PuppeteerSharp.Tests/CSSCoverageTests/CSSCoverageTests.cs]: remove this?

File: lib/PuppeteerSharp.Tests/JSCoverageTests/JSCoverageTests.cs [hunk 2]
```diff
@@ -3,6 +3,7 @@
 using System.Text.RegularExpressions;
 using System.Threading.Tasks;
 using Newtonsoft.Json;
+using Newtonsoft.Json.Serialization;
```
Comment (Meir017) [inline lib/PuppeteerSharp.Tests/JSCoverageTests/JSCoverageTests.cs]: remove this?

File: lib/PuppeteerSharp.Tests/PageTests/EvaluateTests.cs [hunk 3]
```diff
@@ -1,10 +1,9 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
+﻿using System.Collections.Generic;
```
Comment (Meir017) [inline lib/PuppeteerSharp.Tests/PageTests/EvaluateTests.cs]: remove this?

File: lib/PuppeteerSharp.Tests/PageTests/EvaluateTests.cs [hunk 4]
```diff
@@ -1,10 +1,9 @@
...
 using System.Threading.Tasks;
 using Newtonsoft.Json.Linq;
 using Xunit;
 using Xunit.Abstractions;
+using PuppeteerSharp.Helpers;
```
Comment (Meir017) [inline lib/PuppeteerSharp.Tests/PageTests/EvaluateTests.cs]: remove this?

File: lib/PuppeteerSharp.Tests/TargetTests/CreateCDPSessionTests.cs [hunk 5]
```diff
@@ -1,11 +1,10 @@
 ﻿using System;
 using System.Collections.Generic;
-using System.Linq;
-using System.Text;
 using System.Threading.Tasks;
 using Newtonsoft.Json.Linq;
 using Xunit;
 using Xunit.Abstractions;
+using PuppeteerSharp.Helpers;
```
Comment (Meir017) [inline lib/PuppeteerSharp.Tests/TargetTests/CreateCDPSessionTests.cs]: remove this?

File: lib/PuppeteerSharp/CDPSession.cs [hunk 6]
```diff
@@ -147,7 +148,11 @@ public async Task<JObject> SendAsync(string method, dynamic args = null, bool wa
                 { MessageKeys.Id, id },
                 { MessageKeys.Method, method },
                 { MessageKeys.Params, args }
+            }, new JsonSerializerSettings
```
Comment (Meir017) [inline lib/PuppeteerSharp/CDPSession.cs]: we should create a singleton for the json-settings

File: lib/PuppeteerSharp/CookieParam.cs [hunk 7]
```diff
@@ -14,61 +14,59 @@ public class CookieParam
...
         /// </summary>
         /// <value>The value.</value>
-        [JsonProperty("value")]
         public string Value { get; set; }
         /// <summary>
         /// Gets or sets the domain.
         /// </summary>
         /// <value>The domain.</value>
-        [JsonProperty("domain", NullValueHandling = NullValueHandling.Ignore)]
+        [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
```
Comment (Meir017) [inline lib/PuppeteerSharp/CookieParam.cs]: maybe use the `NullValueHandling = NullValueHandling.Ignore` in the json-settings instead?

File: lib/PuppeteerSharp.Tests/PageTests/EvaluateTests.cs (hunk 3 above)
Comment (kblok) [reply lib/PuppeteerSharp.Tests/PageTests/EvaluateTests.cs]: We are using this guy

File: lib/PuppeteerSharp/Helpers/JsonHelper.cs [hunk 8]
```diff
@@ -0,0 +1,14 @@
...
+using Newtonsoft.Json;
+using Newtonsoft.Json.Serialization;
+
+namespace PuppeteerSharp.Helpers
+{
+    internal static class JsonHelper
+    {
+        public static JsonSerializerSettings DefaultJsonSerializerSettings = new JsonSerializerSettings
+        {
+            ContractResolver = new CamelCasePropertyNamesContractResolver()
```
Comment (Meir017) [inline lib/PuppeteerSharp/Helpers/JsonHelper.cs]: shouldn't this include `NullValueHandling = NullValueHandling.Ignore`?

File: lib/PuppeteerSharp/Helpers/JsonHelper.cs [hunk 9]
```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Serialization;
+
+namespace PuppeteerSharp.Helpers
+{
+    internal static class JsonHelper
+    {
+        public static JsonSerializerSettings DefaultJsonSerializerSettings = new JsonSerializerSettings
```
Comment (Meir017) [inline lib/PuppeteerSharp/Helpers/JsonHelper.cs]: I think this should be readonly

File: lib/PuppeteerSharp/Helpers/JsonHelper.cs (hunk 8 above)
Comment (kblok) [reply lib/PuppeteerSharp/Helpers/JsonHelper.cs]: Did that, next commit :)

File: lib/PuppeteerSharp/Helpers/JsonHelper.cs (hunk 9 above)
Comment (kblok) [reply lib/PuppeteerSharp/Helpers/JsonHelper.cs]: done

**Comment by kblok** [comment]
Ready to review @Meir017. Had to revert the Handling null changes, because many tests failed because Chromium was expecting getting nulls

**Pull request merged**